gsa/src/gmp/models/note.js

```diff
@@ -68,6 +93,10 @@ class Note extends Model {
 
     return ret;
   }
+
+  isInTrash() {
+    return this.trash === 1;
```

Comment:
should be YES_VALUE

Reply:
I'll have to make another PR for this, unfortunately. Let me know if there is something else you would like to have changed.